src/System.Runtime.Extensions/src/System/IO/Path.Unix.cs

```diff
@@ -62,6 +60,8 @@ public static string GetFullPath(string path)
 
             return result;
         }
+        
+        public static char[] GetInvalidFileNameChars() => new char[] { '\0', '/' };
```

Comment:
Maybe this should be closer to the top of the file under the other constant/field definitions.